tools/cli_helper.class.inc

```diff
@@ -0,0 +1,181 @@
+<?php declare(strict_types=1);
```

Comment:
This file has its own PR #5968. It's not necessary here but I find i helpful.

Reply:
added specific comment in #5968